### DISCUSSION TITLE
How do I re-use an assignment?

### DISCUSSION THREAD

**Discussion by nashnj13** [community/community#112272] [GitHub Education]
### Hi everyone,

Hi, I am trying to utilize the 're-use assignment' feature on git classroom, but I am unable to edit or share the new assignment with students. 

After selecting re-use assignment, the assignment is added, but shows as inactive:
![image](https://github.com/community/community/assets/102609656/687c26bd-a36b-417f-93c5-d178cf2e5ef8)

When I try to edit the assignment I get a 500 error and when I try to go to the assignment invite link I get this error:
![image](https://github.com/community/community/assets/102609656/43cfad6b-13d6-4fd4-b230-a47abe5254f4)

Am I doing something wrong or is this feature just currently broken?

**Comment by serefakbogfa** [comment]
Check Permissions: Make sure you have the necessary permissions to edit and share assignments in Git Classroom. Ensure that you are logged in with the correct account and that you have the appropriate role (e.g., instructor, administrator) that grants you these permissions.

**Reply by serefakbogfa** [reply]
#g

**Reply by nashnj13** [reply]
I do have the necessary permission to edit and share assignments and I am logged in under the correct account. To give more information, I have noticed that I also get an error when I return to the main classroom page.
![image](https://github.com/community/community/assets/102609656/9501f8ff-1f03-4330-b717-e4a93adfa0ef)
![image](https://github.com/community/community/assets/102609656/519888e3-c38c-4866-9588-b34b8596c749)
